CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 - Fluent EVM API definition for Tangerine Whistle had incorrect code size validation configured [#6382](https://github.com/hyperledger/besu/pull/6382)
 - Correct mining beneficiary for Clique networks in TraceServiceImpl [#6390](https://github.com/hyperledger/besu/pull/6390)
 - Fix to gas limit delta calculations used in block production. Besu should now increment or decrement the block gas limit towards its target correctly (thanks @arbora) #6425
+- Fix the way an advertised host configured with `--p2p-host` is treated when communicating with the originator of a PING packet [#6225](https://github.com/hyperledger/besu/pull/6225)
```

Comment:
this should go in a 24.1.2-SNAPSHOT block, would you mind creating that and moving this into it?

Reply:
Sure, I'll do that now

Reply:
Should be resolved now @garyschulte